frontend/plugins/custom-directives/tooltip.directive.js

```diff
@@ -82,6 +95,25 @@ Vue.directive("tooltip", {
         tooltip.style.display = "none";
       }
     };
+    element.scrollInParent = function () {
+      const { top: parentOffsetTop, bottom: parentOffsetBottom } =
+        getScrollableParent(element).getBoundingClientRect();
+      if (
+        elementOffset.top < parentOffsetTop ||
+        elementOffset.bottom > parentOffsetBottom
+      ) {
+        tooltip.style.visibility = "hidden";
+      } else {
+        tooltip.style.visibility = "visible";
+      }
+      elementOffset = element.getBoundingClientRect();
```

Comment:
why do we need to init two times `elementOffset = element.getBoundingClientRect();` ?
If it's necessarry, put this in a function call initElementOffset()